client/src/components/NewUser.jsx

```diff
@@ -24,39 +22,32 @@ class NewUser extends React.Component {
   handleInputChange(event) {
     const value = event.target.value;
     const name = event.target.name;
-
-    // When a user types into any field field
-    // update the field and reset all duplicate username
-    // and invalid input errors (probably not best behavior ....)
-
-    this.setState({
-      [name]: value, 
-      duplicateUsername: false,
-      invalidInput: false
-    });
+    if (name === 'pictureUrl' && value === '') {
+      this.setState({
+        pictureUrl: '/images/profile_default.jpg'
+      })
```

Comment:
I think it's probably a bad idea for us to be storing the default image URL into the database. If we ever move the location of our default image, or if we want to change the default, these would be locked into the database.

A better approach would be to store null value for the pictureURL DB column for these users, and then show the default as a fallback option for any user with null value set for this field

However -- I suspect this behavior was already happening when we got legacy, so I'm fine shipping as is.

Reply:
It was happening this way already, I just added the ability to add your own picture instead of the default on sign up. I will add a ticket to trello for changing that later.